akka-stream-tests/src/test/scala/akka/stream/scaladsl/SourceWithContextSpec.scala

```diff
@@ -21,8 +21,8 @@ class SourceWithContextSpec extends StreamSpec {
       val msg = Message("a", 1L)
       Source(Vector(msg))
         .asSourceWithContext(_.offset)
-        .asSource
-        .runWith(TestSink.probe[(Message, Long)])
+        .toMat(TestSink.probe[(Message, Long)])(Keep.right)
+        .run
```

Comment:
Is there still test coverage of `asSource` elsewhere?

Reply:
There is "be able to get turned back into a normal Source"